sessions/styled-components-exercise/src/Exercises/ExerciseOne/index.jsx

```diff
@@ -0,0 +1,71 @@
+import styled, { css } from 'styled-components';
+/*
+EXERCISE 1
+
+[x] Create a button as styled-component and call it "StyledButton"
+[x] Think of at least 3 variants for this button
+[ ] Each variant should have its own
+    [ ] color
+    [ ] background
+    [ ] border
+    [ ] hover
+[ ] Choose styles for a disabled version of one of your created button variants
+[ ] Decide for one of your buttons variants as default
+[ ] Move StyledButton to a separate file and import it.
```

Comment:
I am not really sure what you mean. I get that I could change "Think of" to "Implement" but you also removed the checkboxes? This is by far the most important part of the task and I strongly disagree with that.

Reply:
I meant only the checkboxes, even though we could think about implementing them in all exercises. The goal is to keep the exercise text design as homogeneous as possible (which is horrible at the moment ^^). WIth the checkboxes we get jet another way to write the exercises down. We should get together and define a standard way how to design these texts.

Reply:
I would be in favour of establishing a style that is close to our agile working methods and promotes feature-based thinking. Just as we would write acceptance criteria/tasks for our user-stories. 

That is why I decided for checkboxes as a conceptional idea for the tasks I wrote. I received pretty good feedback for this task-style.

Reply:
I would like to see them implemented as well, I just remember that Jerry  said to me once that we shouldnt implement new versions ourself, but rather discussing them with everyone so that we can establish a standard. So my suggestion is that we stick with the old styling and implement the checkboxes as soon as we agreed on a standard, which might include even more changes next to the checkboxes. Otherwise we have to rewrite the texts of all exercises over and over again for each little change that we propose.

Reply:
This task works with checking up your todos. The first two todos are already checked to show that they are already done. Removing the checkboxes will break the concept of this particular exercise.

I would prefer letting this PR on hold until this concept is practicable, instead of merging a broken exercise.